pkg/sysfs/system.go

```diff
@@ -30,9 +30,12 @@ import (
 	idset "github.com/intel/goresctrl/pkg/utils"
 )
 
+var (
+	// Sysfs root directory.
+	sysfsRoot = ""
```

Comment:
nit: I think this is misleading here as it kinda represents the "host" root mount point. Rename to `sysRoot` or `hostRoot`, WDYT?

Reply:
Okay, true. I'll update.

Reply:
Updated taking that comment into account, and therefore also changing the function names to `SetSysRoot()` as opposed to the original `SetSysfsRoot()`.